demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -49,6 +51,8 @@ def __init__(self, pack_path: str, update_type: Union[str, None], modified_files
         self.should_delete_existing_rn = False
         self.pack_metadata_only = pack_metadata_only
         self.is_force = is_force
+        self.git_util = GitUtil(repo=Content.git())
```

Comment:
why do you need to save the `git_util` object? it's not being used elsewhere

Reply:
https://github.com/demisto/demisto-sdk/pull/1758/commits/9f722bc6cf8be951ecad4cba2c801dae1d1b0cef